orte/mca/grpcomm/grpcomm.h

```diff
@@ -56,6 +56,8 @@ BEGIN_C_DECLS
  * collective completion */
 typedef void (*orte_grpcomm_cbfunc_t)(int status, opal_buffer_t *buf, void *cbdata);
 
+typedef int (*orte_grpcomm_rbcast_cb_t)(opal_buffer_t* buffer);
```

Comment:
This callback seems to lack some information, such as error and cb_data. Why  it is not the same as above ?

Reply:
Because i just need to pass this message to propagator, i am not sure i need error and cb_data.